Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM ubuntu:18.04
+MAINTAINER Canonical
```

Comment:
Who is Canonical?

Reply:
They are the Ubuntu official maintainer.

Reply:
He is not relevant for a layered image. You should put your name.